src/main.ts

```diff
@@ -50,3 +50,27 @@ app.directive("focus", {
     el.stop();
   },
 });
+
+// standard html email validation is not sufficient
+const emailregex =
+  /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
I would assume that a browser holds a more robust regex for validating e-mails than that this probably is. Feels as if we're trying to reinvent the wheel here and it seems hard to manage (I have no clue what this regex says, to be fair I know little to none about regex).

Above being said, it does seem to do what is requested in the test feedback.